tests/explorer.api.tests/ApiTests.cs

```diff
@@ -166,7 +171,7 @@ public async Task SuccessWithResult()
         public async Task FailWithBadEndPoint(string endpoint)
         {
             await TestApi(HttpMethod.Post, endpoint, ValidData, test: (response, content) =>
-                Assert.True(response.StatusCode == HttpStatusCode.NotFound, content));
+                Assert.False(response.IsSuccessStatusCode, content));
```

Comment:
The original assert contained additional information about the status code.

Reply:
Yeah this was a bit lazy. I did it because the api extension returns a BadResponse (400) or NotAllowed (405) instead of NotFound (404). So depending on whether the test case hits the api version checker or our api endpoint we see different response codes. This was just a convenient catch-all. 

I suppose I shouldn't be testing the api versioning here (that's an external package after all and I assume it's tested by Microsoft 😉). I'll add the `/api/v1` prefix to the test urls.

Reply:
I've made the tests more specific: 959ea03